src/calendar/day/basic/index.tsx

```diff
@@ -8,7 +8,29 @@ import {shouldUpdate} from '../../../component-updater';
 import styleConstructor from './style';
 import Marking from '../marking';
 
-export default class BasicDay extends Component {
+interface BasicDayProps {
```

Comment:
Don't we need to export the props for the components that use them, like Calendar (line #31: `...Day.propTypes`)?

Reply:
Well yea, we can also do it when we migrate the Calendar component to typescript...